includes/cli/class-data-backfill.php

```diff
@@ -0,0 +1,362 @@
+<?php
+/**
+ * Data Backfill scripts.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network;
+
+use Automattic\Jetpack\VideoPress\Data;
```

Comment:
left over

Reply:
Fixed in 111a7b4fde706d7f6de42f2237ee619e8a36b477